vita3k/modules/SceGxm/SceGxm.cpp

```diff
@@ -636,15 +733,21 @@ EXPORT(int, sceGxmGetParameterBufferThreshold) {
 }
 
 EXPORT(int, sceGxmGetPrecomputedDrawSize) {
-    return UNIMPLEMENTED();
+    STUBBED("handle");
+
+    return 4;
 }
 
 EXPORT(int, sceGxmGetPrecomputedFragmentStateSize) {
-    return UNIMPLEMENTED();
+    STUBBED("handle");
```

Comment:
technically not stubbed anymore

Reply:
should i remove the stubs

Reply:
Why not. Set the other two sizes to 0 btw.